src/ballet/hex/fd_hex.c

```diff
@@ -0,0 +1,31 @@
+#include "fd_hex.h"
+
+/* FIXME use LUT instead? */
```

Comment:
My preference is not to leave these FIXME and TODO that will never get fixed. Can just remove them, will be obvious and get addressed if it's a bottleneck.